chains/sonic.json

```diff
@@ -0,0 +1,17 @@
+{
+  "alias": "sonic",
+  "decimals": 18,
+  "explorer": {
+    "browserUrl": "https://explorer.soniclabs.com/"
```

Comment:
Not working for now. We can check it later again.

Reply:
Seems like it works now along with testnet explorer

Reply:
Still not working for me. Tried with different DNS, browser, VPN stuff.